articles/connections/references/apple-native/guides/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Add Sign In with Apple to Your Native App
+description: Learn how to add native login functionality to your native app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Your Native App
+
+To set up and configure Sign In with Apple for your native app, you will do the following:
+
+1. Have an [Apple Developer](https://developer.apple.com/programs/) account, which is a paid account with Apple. (There is no free trial available unless you are part of their [iOS Developer University Program](https://developer.apple.com/support/compare-memberships/)).
+2. [Register your app in your Apple Developer account](#register-your-app-in-your-apple-developer-account)
+3. [Configure the application connection in Auth0](#configure-the-connection-in-auth0)
+
+## Register your native app on your Apple Developer account
+
+When setting up your application with your Apple Developer account, make sure you save the following IDs and keys for the application connection settings in the Auth0 Dashboard:
+
+* Client ID (the Service ID)
+* Client Secret Signing Key
+* Apple Team ID (App ID)
+* Client Signing Key ID (optional)
+
+1. When you sign into the developer account, go to the [Certificates, IDs, & Profiles](https://developer.apple.com/account/resources/certificates/list) section, and choose the [Identifiers](https://developer.apple.com/account/resources/identifiers/list/serviceId) subsection on the left menu.
+
+2. Click **Register an App ID** button.
+
+3. Choose **App IDs** as the identifier type, and click **Continue**. 
+
+4. Enter a description for your new App ID and a Bundle ID. For the latter, Apple recommends using a reverse-domain name style string (for example, `com.<YOUR CUSTOM DOMAIN>.appid`). 
+
+5. Scroll down and check the **Sign In with Apple** feature. You won't have to use the **Edit** here. You will configure this feature later.
+
+6. Leave the other options with their default values and click **Continue**. When you click on this button, Apple  displays a summary of the options you just configured. If everything is correct, click **Register**. Apple redirects you to the **Certificates, Identifiers & Profiles** subsection again, this time listing your new App ID. 
+
+7. Create a Service ID that represents your application. This might look like a redundant effort, but Apple organizes it so you can nest multiple Services IDs under the same App ID. This makes sense when you have distinct versions of your application to support different devices.
```

Comment:
This step is not strictly necessary to get an iOS app running.  Do we need to include it in a getting started style guide?